packages/app-js/src/Intro.tsx

```diff
@@ -0,0 +1,30 @@
+// Copyright 2017-2019 @polkadot/app-js authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { I18nProps } from '@polkadot/ui-app/types';
+
+import { Trans } from 'react-i18next';
+import React, { Fragment } from 'react';
+
+import translate from './translate';
+
+type Props = I18nProps;
+
+const Intro = (props: Props) => {
+  const { t } = props;
+  return (
+    <Fragment>
```

Comment:
Prefer a simple `<>` (with terminating `</>`) - no need to import `Fragment`

Reply:
According to the React docs `<></>` is just syntactic sugar for `<React.Fragment />`, not the other way round.
https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html

> n React, this desugars to a <React.Fragment/> element, as in the example from the previous section.

Reply:
100%, is is syntactic sugar only and equivalent - which also means we don't need to import `Fragment`, we can just use it.

Reply:
I prefer using `<Fragment>` directly because I don't really like the empty html tag syntax. But since this is purely down to personal preferences, I'm happy to change it to `<></>`  – your project, your preferences.

Reply:
...and when I'm removing the `<p></p>` because we're not exposing the api here in apps but in the api itself, a wrapper is not needed anyways.

Reply:
Yes, this one it is a bit of bike-shedding, no doubt. I'm seemingly alone in my love for the "ugly" version :)